src/app/earn/components/earn-widget/components/base-token-selector.tsx

```diff
@@ -0,0 +1,23 @@
+import { ChevronDownIcon } from '@chakra-ui/icons'
+import Image from 'next/image'
+
+import { Token } from '@/constants/tokens'
+
+type BaseTokenSelectorProps = {
+  baseToken: Token
+  onClick: () => void
+}
+
+export function BaseTokenSelector(props: BaseTokenSelectorProps) {
```

Comment:
nit: any reason not to destructure here?

Reply:
@tokdaniel copied the existing code and just pulled it out to the 'higher' directory, though I went ahead and destructured this one